src/libs/antares/study/parts/hydro/finallevelinflowsmodifyer.cpp

```diff
@@ -169,7 +182,7 @@ bool FinalLevelInflowsModifier::isApplicable(uint year)
 {
     // If isApplicable_.size() == 0, then instance was not properly initialized
     // and is not applicable.
-    return isActive() && isApplicable_.size() && isApplicable_.at(year);
+    return isApplicable_.size() && isApplicable_.at(year);
```

Comment:
Both versions are correct, though using `empty` makes it a bit more clear
```suggestion
    return !isApplicable_.empty() && isApplicable_.at(year);
```